cmd/protoc-gen-openapi/generator/generator.go

```diff
@@ -272,7 +272,7 @@ func (g *OpenAPIv3Generator) findAndFormatFieldName(name string, inMessage *prot
 // or a repeated primitive type or a non-repeated message type.
 // In the case of a repeated type, the parameter can be repeated in the URL as ...?param=A&param=B.
 // In the case of a message type, each field of the message is mapped to a separate parameter,
-// such as ...?foo.a=A&foo.b=B&foo.c=C.
+// such as ...?foo.a=A&foo.b=B&foo.c=C. Except for google.protobuf.timestamp type it will be serialized as a string
```

Comment:
I think there is a lot of exeption based on this guide https://developers.google.com/protocol-buffers/docs/proto3#json

Reply:
including wrappers type that i made

Reply:
I think we should put it in your PR then mas @galihputera ?

I'll create a PR in upstream but with my changes only (without your changes), because your PR in upstream is not merged yet right ?

Reply:
for PR in this fork, I already add the comment @galihputera